Mondee/MondeeApp.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct MondeeApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            ReceiveDataSampleView()
```

Comment:
ContentView로 수정 부탁드립니다.
```suggestion
            ContentView()
```